### PR TITLE
.NET Standard 2.1 nuget package configuration

### PR DESCRIPTION
**Pull request by Akarinnnnn**
For #184 , Ready to use.
Once ready to merge this pull request, I will transfer those packages' ownership back to you.

### CODE REVIEW

**Comment by Akarinnnnn** [comment]
I need your Nuget account name to add owner.

File: Standalone/Steamworks.NET.Standard.csproj [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
+    <PackageId>Steamworks.NET.Standard.$(Configuration).$(platform)</PackageId>
+    <Authors>rlabrecque</Authors>
+    <PackageVersion>14.0.1.2</PackageVersion>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/rlabrecque/Steamworks.NET</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/rlabrecque/Steamworks.NET.git</RepositoryUrl>
+  </PropertyGroup>
+    
+  <ItemGroup>
+    <Compile Remove="%2a%2a\**" />
```
Comment (yaakov-h) [inline Standalone/Steamworks.NET.Standard.csproj:24]: This seems dodgy... mixing URL-escaped characters with unescaped ones?
AFAIK they should all be unescaped.

**Comment by rlabrecque** [comment]
https://www.nuget.org/profiles/rileylabrecque

File: Standalone/Steamworks.NET.Standard.csproj (hunk 1 above)
Comment (Akarinnnnn) [reply Standalone/Steamworks.NET.Standard.csproj:24]: I don't know why, remove or edit this will cause complication failure. So I deceided to keep it.

**Pull request merged**